assets/i18n/en.json

```diff
@@ -562,26 +565,25 @@
-    "I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_6": "<strong>Stay interesting!</strong> - The best explorations tell a story, and have useful feedback. They give learners a chance to reason about concepts, try out new knowledge, and receive useful feedback on their work.",
-    "I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_PARAGRAPH_1": "When an exploration is published, it is credited to you, and becomes available for anyone to learn from. In order to create a good learning experience for students, here's a few tips we've found useful:",
-    "I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_PARAGRAPH_2": "For additional help on crafting useful feedback and good explorations, check out our <a href=\"http://oppia.github.io/#/DesignTips\" class=\"inline-links\">Design Tips</a> page.",
-    "I18N_TEACH_PAGE_PUBLICATION_POLICY_HEADING": "Publication policy",
-    "I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_1": "Any owner of a private exploration can publish it at any time. However, low-quality explorations may be unpublished by the moderators, with feedback on how to improve them.",
-    "I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_2": "Explorations are meant to be continuously improved, and explorations that haven't been edited for a long time may be considered \"orphaned\". In this case, ownership may be handed to the Oppia community at large (at moderator discretion), so that the exploration can continue to be improved.",
-    "I18N_TEACH_PAGE_TAB_PARTICIPATION_PLAYBOOK": "Participation Playbook",
-    "I18N_TEACH_PAGE_TAB_TEACH_WITH_OPPIA": "Teach with Oppia",
+    "I18N_TEACH_BENEFITS_TITLE": "Our Benefits",
+    "I18N_TEACH_PAGE_ACTION_START_LEARNING": "Start Learning",
+    "I18N_TEACH_PAGE_CLASSROOM_BUTTON": "VISIT CLASSROOM",
+    "I18N_TEACH_PAGE_CLASSROOM_CONTENT": "In the Classroom, you can find a set of lessons that the Oppia team has designed and tested to make sure they’re effective and fun for all learners. All of the lessons have been reviewed by teachers and experts, so you can feel assured that your students are getting an effective education, while learning at their own pace.",
+    "I18N_TEACH_PAGE_CLASSROOM_TITLE": "Learn from Oppia’s Tested and Verified Lessons",
+    "I18N_TEACH_PAGE_CONTENT" : "Oppia is an engaging, new approach to online learning that specifically is made to ensure that everyone has access to quality education.",
+    "I18N_TEACH_PAGE_HEADING": "Oppia for Parents, Teachers and Gaurdians",
+    "I18N_TEACH_PAGE_LIBRARY_BUTTON": "BROWSE LIBRARY",
+    "I18N_TEACH_PAGE_LIBRARY_CONTENT": "Educators and community members around the world use Oppia’s lesson creation platform as a way to create and share lessons. You can find over 20,000 lessons for 17 different subjects in our Exploration Library, and maybe you’ll be inspired to create your own!",
+    "I18N_TEACH_PAGE_LIBRARY_TITLE": "Explore Lessons Made By the Community",
+    "I18N_TEACH_PAGE_SIX_TITLE": "Get Started with Curated Tips",
```

Comment:
Until we actually publish our curated tips, we should adjust this title to more accurately reflect the fact that we only have a CTA underneath that links to our lessons. I suggest changing to "Get Started Learning Today"

Reply:
> * The Library preview image should have a larger margin on the left and right. It should be consistent with other images, text, etc.

Mam, can you tell which view exactly, I think it is aligned in the desktop view with the images .
![Screenshot from 2021-01-13 09-34-12](https://user-images.githubusercontent.com/68547101/104405301-a5c20f00-5582-11eb-9d1e-1ec8222fa392.png)

Reply:
Sorry in the mobile view specifically. The shadow and frame helps a bit but it does feel as though it's a bit too close to the edges that it's a bit uncomfortable. @mschanteltc - Are there guidelines for how much space we should have on the sides? 
<img width="265" alt="Screen Shot 2021-01-13 at 6 51 07 PM" src="https://user-images.githubusercontent.com/12034267/104538237-586c8e80-55d0-11eb-9de7-c7a4d0d89ea5.png">

Reply:
mam does this look okay?
![Screenshot from 2021-01-14 11-44-34](https://user-images.githubusercontent.com/68547101/104552206-5c46f200-565e-11eb-85f1-c580418d1a9a.png)